mm2src/mm2_main/src/lp_swap/recreate_swap_data.rs

```diff
@@ -126,7 +126,7 @@ fn recreate_maker_swap(ctx: MmArc, taker_swap: TakerSavedSwap) -> RecreateSwapRe
         // We could parse the `TakerSwapEvent::TakerPaymentSpent` event.
         // As for now, don't try to find the secret in the events since we can refund without it.
         secret: H256Json::default(),
-        secret_hash: Some(negotiated_event.secret_hash),
+        secret_hash: Some(negotiated_event.secret_hash.0[..].into()),
```

Comment:
Why can't this be just
```rust
secret_hash: Some(negotiated_event.secret_hash)
```

Reply:
Made this before `negotiated_event.secret_hash` became `BytesJson`... Will fix!

Reply:
Done, thanks!